internal/fs/vss_windows.go

```diff
@@ -767,6 +792,43 @@ func NewVssSnapshot(volume string, timeoutInSeconds uint) (VssSnapshot, error) {
 		)
 	}
 
+	mountPoints, err := enumerateMountedFolders(volume)
+	if err != nil {
+		return VssSnapshot{}, fmt.Errorf(
+			"VSS error: failed to enumerate mount points for volume %s: %s",
+			volume, err,
+		)
+	}
+
+	mountPointInfo := make(map[string]MountPoint)
+
+	for _, mountPoint := range mountPoints {
+		// ensure every mountpoint is available even without a valid
+		// snapshot because we need to consider this when backing up files
+		mountPointInfo[mountPoint] = MountPoint{
+			isSnapshotted: false,
+		}
+
+		if hRes, isSupported := iVssBackupComponents.IsVolumeSupported(mountPoint); hRes != S_OK {
+			continue
+		} else if !isSupported {
+			continue
+		}
+
+		var mountPointSnapshotSetID ole.GUID
+		if hRes := iVssBackupComponents.AddToSnapshotSet(mountPoint, &mountPointSnapshotSetID); hRes != S_OK {
+			return VssSnapshot{}, fmt.Errorf(
```

Comment:
I don't think fail-fast design is good here: in practice this specific mountpoint may not be used at all.
Just log error and continue.

Reply:
It should not fail here at all because I've already tested VSS is supported. If it fails here there is some reason for it.